cpp/src/gandiva/gdv_function_stubs.h

```diff
@@ -69,8 +69,18 @@ const char* gdv_fn_regexp_extract_utf8_utf8_int32(
     int64_t ptr, int64_t holder_ptr, const char* data, int32_t data_len,
     const char* /*pattern*/, int32_t /*pattern_len*/, const int32_t idx, int32_t* out_length);
 
-const char* gdv_fn_substr_index_utf8_utf8_int32(int64_t ptr, int64_t holder_ptr, const char* input, int in_len,
-                                         const char* delim, int delim_len, int count, int32_t* out_len);
+const char* gdv_fn_parse_url_utf8_utf8(
```

Comment:
this func def should be removed, otherwise the building failed
fixed #139

Reply:
Thanks for your fix.